src/scrubber.ts

```diff
@@ -160,8 +161,13 @@ export class Scrubber {
     if (!a || !b) return false
     if (a === b) return true
 
+    // Remove any entries that are integers as we assume they are array indices that should be ignored for parent matching
+    let aSliced = a.filter(e => !e.match(/^[0-9]*$/g))
+
+    if (a.length < b.length) return false
```

Comment:
More performant to put this check above the regex-check

Reply:
Maybe do it twice then since I added it as a safety check in case regex filter reduces the size of a to the point of a being smaller than b.

Reply:
Although it's checking the wrong array, should be sliced one